scripts/asym_crypto_yaml.py

```diff
@@ -45,11 +46,20 @@ def add_secret(yaml_key, secret_contents, public_key_file, target_secret_file):
 def decrypt_encrypted_yaml(secrets_file_path, private_key_path, outfile_path):
     decrypt_yaml_file_and_write_encrypted_file_to_disk(secrets_file_path, private_key_path, outfile_path)
 
+
+@click.command()
+@click.option('--secrets_file_path', help='', default="config/config.yaml")
+@click.option('--private_key_path', help='')
```

Comment:
Can you fill out the cli help bits here, and make it clear this is the old private key and the new public key?